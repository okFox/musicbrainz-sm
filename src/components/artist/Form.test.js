import React from 'react';
import { shallow } from 'enzyme';
import ArtistInput from './Form';

describe('ArtistInput component', () => {
  it('renders ArtistInput', () => {
    const wrapper = shallow(<ArtistInput onSubmit={()=>{}} onChange={()=>{}} artistName="name" />);
    expect(wrapper).toMatchSnapshot();
  });
});
