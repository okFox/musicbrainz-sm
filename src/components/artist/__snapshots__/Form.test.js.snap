// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistInput component renders ArtistInput 1`] = `
<form
  className="form"
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    type="text"
    value="name"
  />
  <button>
    Search!
  </button>
</form>
`;
